Gemfile

```diff
@@ -65,6 +65,7 @@ group :development, :test do
   gem 'cucumber-rails', :require => false
   gem 'test-factory', '~> 0.5.3'
   gem 'faker', '1.4.2'
+  gem 'rubocop'
```

Comment:
The README for Rubocop advises to lock to a particular version because [they have a tendency to make backwards-incompatible changes](https://github.com/bbatsov/rubocop#installation). So, this should probably change to:
```ruby
gem 'rubocop', '~> 0.39.0', require: false
```